src/domain/event/eventInfo/otherEventTimes/OtherEventTimes.tsx

```diff
@@ -0,0 +1,106 @@
+import { IconAngleDown, IconArrowRight, IconCalendarPlus } from 'hds-react';
```

Comment:
Files seems to be just moved but git shows it as new file which makes reviewing harder. `git mv` or just `mv` file then git will show it as moved file.

Reply:
oh, never used those. Good to know in future